### DISCUSSION TITLE
How to place json object array into a map

### DISCUSSION THREAD

**Discussion by HFDan** [nlohmann/json#2733] [Q&A] [answered]
so i have this json:
```json
{
    "projects": [
        {
            "name": "aaa",
            "language": "bbb",
            "path": "ccc/ddd"
        },
        {
            "name": "eee",
            "language": "fff",
            "path": "ggg/hhh"
        }
    ]
}
```

And the `std::map` looks like this: `std::map<std::string, project>`, where `project` is a custom struct.

I would like the "name" value of the json object to go into the `std::string` of the map (the key), and the "language" and "path" to go into the `project` struct (the value).

Is this possible?

**Comment by nlohmann** [comment, marked as answer]
The [`items()`](https://json.nlohmann.me/api/basic_json/items/) function should be helpful here.

**Reply by HFDan** [reply]
> The [`items()`](https://json.nlohmann.me/api/basic_json/items/) function should be helpful here.

Thank you, i will check it out!
